docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,47 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 10 May 2023
+
+Frontend version: .0.0-20230509.1043 | Backend version: 1.61.9-230508.0923
```

Comment:
FE version typo, should be 1.0.0-20230509.1043

Reply:
fixed